src/wp-admin/includes/class-wp-automatic-updater.php

```diff
@@ -131,7 +131,29 @@ public function is_vcs_checkout( $context ) {
 	 *
 	 * @param string $type    The type of update being checked: 'core', 'theme',
 	 *                        'plugin', 'translation'.
-	 * @param object $item    The update offer.
+	 * @param object $item {
+	 *     The update offer.
+	 *
+	 *     @type string   $current            The version number of the update offer.
+	 *     @type bool     $disable_autoupdate Whether automatic updates for this particular update offer have been
+	 *                                        remotely disabled via the update API response.
+	 *     @type bool     $dismissed          Whether this update offer has previously been dismissed.
+	 *     @type string   $download           URL to the ZIP file for the update.
```

Comment:
For Core, this always seems to point to the `package['full']` URL. But not clear if this will always be the case.